From 0af388b671d110e3b56b298151a04879c8fe54d9 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Wed, 3 Jul 2019 19:59:10 +0000
Subject: [PATCH] TOOLS-2279 sunset 13.3.1 image building support Reviewed by:
 Tim Foster <tim.foster@joyent.com> Approved by: Tim Foster
 <tim.foster@joyent.com>

---
 README.md | 1 -
 1 file changed, 1 deletion(-)

diff --git a/README.md b/README.md
index 38ed2a1..792eb89 100644
--- a/README.md
+++ b/README.md
@@ -109,7 +109,6 @@ files. We will try to keep this table up to date:
 | triton-origin-multiarch-18.1.0 | minimal-multiarch@18.1.0     | minimal-multiarch@18.1.0         | 1ad363ec-3b83-11e8-8521-2f68a4a34d5d |
 | triton-origin-multiarch-15.4.1 | minimal-multiarch-lts@15.4.1 | sdc-minimal-multiarch-lts@15.4.1 | 18b094b0-eb01-11e5-80c1-175dac7ddf02 |
 | -                              | -                            | sdc-base@14.2.0                  | de411e86-548d-11e4-a4b7-3bb60478632a |
-| -                              | -                            | sdc-multiarch@13.3.1             | b4bdc598-8939-11e3-bea4-8341f6861379 |
 | -                              | -                            | sdc-smartos@1.6.3                | fd2cc906-8938-11e3-beab-4359c665ac99 |
 
 
-- 
2.21.0

